src/js/containers/router/RouterRoutes.jsx

```diff
@@ -70,26 +70,21 @@ const checkUserPermissions = (nextState, replace) => {
     if (session.login != "loggedIn") {
         performAutoLogin(nextState.location, replace);
     }
-    else if (kGlobalConstants.STAGING) {
-        if (session.user.helpOnly || (nextState.routes.length > 1 && nextState.routes[1].type == 'dabs' && !PermissionsHelper.checkDabsReader(session))) {
-            replace('/');
-        }
-    }
-    else if (session.user.helpOnly || (nextState.routes.length > 1 && nextState.routes[1].type == 'dabs' && !PermissionsHelper.checkDabsReader(session))) {
+    else if (session.user.helpOnly) {
         // if no permissions or attempting to reach DABS with improper permissions, bounce to help
         replace('/help');
     }
 }
 
-const checkReadUserPermissions = (nextState, replace) => {
+const checkDabsUserPermissions = (nextState, replace) => {
```

Comment:
If we're only using this in one place (checking if they can start a submission basically). Maybe this should be called `checkDabsUploadPermissions` because the name is misleading. I had to do a double-take when I was reading it because I was confused why we weren't allowing read-only users to get through the DABS permission check.